src/main/java/mate/academy/bookshop/dto/category/CategoryRequestDto.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.bookshop.dto.category;
+
+import jakarta.validation.constraints.NotBlank;
+
+public record CategoryRequestDto(
+        @NotBlank(message = "can not be blank")
```

Comment:
```suggestion
        @NotBlank
```
It has the same default value.

Reply:
Not resolved